data/assets.json

```diff
@@ -6874,6 +6874,108 @@
     "type": "native"
   },
   {
+    "coingecko": "tether",
+    "description": "Tether from Solana via Picasso",
+    "id": {
+      "osmosis": {
+        "osmosis-1": "ibc/D105950618E47CA2AEC314282BC401625025F80A4F812808DEEBB1941C685575"
```

Comment:
should this be changed to this?
`ibc/0233A3F2541FD43DBCA569B27AF886E97F5C03FC0305E4A8A3FAC6AC26249C7A`

Reply:
Yes, sorry, I copied the Source denom from our asset lists